### PR TITLE
#RI-4289 - Hide recommendations

### PR DESCRIPTION
**Pull request by zalenskiSofteq**
#RI-4289 - Hide recommendations

### CODE REVIEW

File: redisinsight/api/migration/index.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ import { databaseAnalysisRecommendations1674660306971 } from './1674660306971-da
 import { databaseTimeout1675398140189 } from './1675398140189-database-timeout';
 import { databaseCompressor1678182722874 } from './1678182722874-database-compressor';
 import { customTutorials1677135091633 } from './1677135091633-custom-tutorials';
-import { databaseRecommendations1681291260647 } from './1681291260647-database-recommendations';
+import { databaseRecommendations1681542719349 } from './1681542719349-database-recommendations'
```
Comment (AmirAllayarovSofteq) [inline redisinsight/api/migration/index.ts:32]: I guess you forgot semicolon here

File: redisinsight/api/package.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
     "test:api": "ts-mocha --paths -p test/api/api.tsconfig.json --config ./test/api/.mocharc.yml",
     "test:api:cov": "nyc --reporter=html --reporter=text --reporter=text-summary yarn run test:api",
     "test:api:ci:cov": "nyc -r text -r text-summary -r html yarn run test:api --reporter mocha-multi-reporters --reporter-options configFile=test/api/reporters.json && nyc merge .nyc_output ./coverage/test-run-coverage.json",
-    "typeorm:migrate": "cross-env NODE_ENV=production yarn typeorm migration:generate ./migration/migration",
+    "typeorm:migrate": "cross-env NODE_ENV=staging yarn typeorm migration:generate ./migration/migration",
```
Comment (arthosofteq) [inline redisinsight/api/package.json:37]: did you forget to rollback?

**Review by arthosofteq — APPROVED** [review]

File: redisinsight/api/package.json (hunk 2 above)
Comment (zalenskiSofteq) [reply redisinsight/api/package.json:37]: no, let's do migrations from staging

File: redisinsight/api/migration/index.ts (hunk 1 above)
Comment (zalenskiSofteq) [reply redisinsight/api/migration/index.ts]: added

**Pull request merged**